tcf_website/static/landing/landing-page.css

```diff
@@ -0,0 +1,155 @@
+/*!
+ * Start Bootstrap - Landing Page v5.0.8 (https://startbootstrap.com/themes/landing-page)
+ * Copyright 2013-2020 Start Bootstrap
+ * Licensed under MIT (https://github.com/StartBootstrap/startbootstrap-landing-page/blob/master/LICENSE)
+ */
+
+/* Html Classes */
+body {
+  font-family: 'Lato', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: 'Lato', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-weight: 700;
+}
+
+/* Navbar */
+nav.navbar {
+  background-color: white;
+}
+
+/* Masthead */
+header.masthead {
+  position: relative;
+  padding-top: 8rem;
+  padding-bottom: 0rem;
+}
+
+header.masthead h1 {
+  font-size: 2rem;
+}
+
+@media (min-width: 768px) {
+  header.masthead {
+    padding-top: 8rem;
+    padding-bottom: 0rem;
+  }
+  header.masthead h1 {
+    font-size: 3rem;
+  }
+}
+
+/* Showcase */
+.showcase .showcase-text {
+  padding: 3rem;
+}
+
+.showcase .showcase-img {
+  min-height: 30rem;
+  background-size: cover;
+}
+
+@media (min-width: 768px) {
+  .showcase .showcase-text {
+    padding: 7rem;
+  }
+}
+
+/* Features */
+.features-icons {
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+}
+
+.features-icons .card {
+  box-shadow: 1px 1px 2px rgba(0,0,0,0.4);
+}
+.features-icons .card:hover {
+  box-shadow: 1px 1px 4px rgba(0,0,0,0.4);
+}
+
+.features-icons .features-icons-item {
+  max-width: 20rem;
+}
+
+.features-icons .features-icons-item .features-icons-icon {
+  height: 7rem;
+  color: var(--main-color);
+  text-shadow: 10px 10px 25px var(--accent-color);
+}
+
+.features-icons .features-icons-item .features-icons-icon i {
+  font-size: 4.5rem;
+}
+
+.features-icons .features-icons-item:hover .features-icons-icon i {
+  font-size: 5rem;
+}
+
+
+/* About team */
+.about-team {
+  padding-top: 5rem;
+  padding-bottom: 5rem;
+}
+/* About member photo hover animation */
+.about-team .about-pfp {
+    overflow: hidden;
+}
+.about-team .about-pfp img{
+    transition: all 0.5s ease;
+}
+.about-team .about-pfp:hover img{
+    transform: scale(1.1);
+}
+.about-team .about-member{
+    border: 0px;
+    /* box-shadow: 2px 2px 5px var(--secondary-color); */
+}
+.about-team .about-link:hover{
+  box-shadow: 2px 2px 5px var(--main-color);
+}
+
+/* About info box */
+.about-team .about-info {
+  border: 3px solid var(--accent-color);
+  border-radius: 0%;
+}
+.about-team .about-info:hover {
+  box-shadow: 4px 4px 10px var(--accent-color);
+}
+
+
+/* Testimonials */
+.testimonials {
+  padding-top: 4rem;
+  padding-bottom: 4rem;
+}
+
+.testimonials .testimonial-item {
+  max-width: 18rem;
+}
+
+
+/* Feedback */
+.feedback {
+  padding-top: 4rem;
+  padding-bottom: 8rem;
+}
+
+
+/* Footer */
+footer.footer {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  background-color: var(--accent-color);
```

Comment:
hmm, is it better practice to use styling classes in html where you can instead of writing css? I can't decide.

Reply:
That's a good question; I couldn't find too much resources pointing either way
I would say to respect DRY that it depends on where it's used:
If let's say you have several cards that you need to style the same way then it would make sense to use CSS, but if you have a single header or single picture than using existing classes when possible would be better since those styles already exist

Reply:
makes sense, I'll make the change!